src/Tools/Source/RunTests/ITestExecutor.cs

```diff
@@ -13,18 +13,16 @@ namespace RunTests
         internal string DotnetFilePath { get; }
         internal ProcDumpInfo? ProcDumpInfo { get; }
         internal string TestResultsDirectory { get; }
-        internal string? Trait { get; }
-        internal string? NoTrait { get; }
+        internal string? TestFilter { get; }
```

Comment:
As best as I could tell, the trait and notrait properties did not work properly with dotnet test.  They would pass an argument like Trait=LanguageServerProtocol, but per https://docs.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=xunit that isn't the correct format.  So I modified these to be able to just pass in the xunit test filter (which also allowed me to pass in fully qualified names)

Reply:
is it possible to extract this out into anotehr pR?